src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private Figure figure;
```

Comment:
redundant and should be removed

Reply:
here I mean that you should remove 'figure' field. random and colorSupplier should be declared as fields :)